collector/installation.go

```diff
@@ -73,6 +76,26 @@ func (i Installation) Collect(c *CollectorOpts) interface{} {
 		log.Errorf("Failed to get authProviders err=%s", err)
 	}
 
+	if len(i.AuthConfig) > 0 {
+		log.Debug("Collecting Users")
+		userList, err := c.Client.User.List(&nonRemoved)
+		if err == nil {
+			for _, user := range userList.Data {
+				for _, principalID := range user.PrincipalIDs {
+					for providerName, _ := range i.AuthConfig {
+						if strings.Contains(principalID, providerName) {
+							i.Users.Increment(providerName)
+							break // only breaks out of single loop
+						}
+					}
+
+				}
+			}
+		} else {
+			log.Errorf("Failed to get users err=%s", err)
```

Comment:
nit: probably make this failed to list user

Reply:
I'd like to keep it in line with all the other log messages.